src/detections/hooks/use-get-detections.js

```diff
@@ -2,14 +2,12 @@ import { useCallback } from 'react'
 import { useDispatch } from 'react-redux'
 
 import { actions } from 'detections/slices/get'
-import { useContext } from 'detections/context'
+import { useParameters } from 'detections/hooks/use-parameters'
 
 export const useGetDetections = () => {
   const dispatch = useDispatch()
   const { attempt } = actions
-  const { pageIndex, pageSize } = useContext()
+  const dataFetch = { ...useParameters() }
```

Comment:
No need to spread here :)))

Reply:
Also, let's rename this to something like `fetchingInput`

Reply:
Or, even better, just input.

Reply:
Done